app/lookups/dog-breed.js

```diff
@@ -2,7 +2,7 @@ const { Op } = require('sequelize')
 const sequelize = require('../config/db')
 
 const getBreed = async breed => {
-  return await sequelize.models.dog_breed.findOne({
```

Comment:
Not sure why 'await' has been removed. Think it's needed

Reply:
No need to await here as it's returning a Promise

Reply:
If you don't want the await, you might need to create a separate call since my import code uses this method - and as far as I know - needs the 'await' :) (see backlog-function.js line 40 and line 95